tests/e2e/common.exp

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/expect -f
+# Copyright 2017 Mirantis
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+proc connect_ssh {vmname} {
```

Comment:
Thanks for noticing, I agree that it's better to have all e2e tests written in bash rather than a mixture of bash and somewhat obsolete `tcl`. Therefore I've removed all the four `.exp` files and implemented everything in `e2e.sh`. Not sure why `git` is so "smart" to think that `pkg/log/Makefile.am` was created out of `vmchat-short.exp`, though (please just ignore).